### PR TITLE
98324 Fixed Link Localization

### PR DESCRIPTION
**Pull request by CurtisUnderwood**
### Description

List of proposed changes:

- Middleware used to ensure English links are defaulted to '/en/'
- All French links are accessible in French ex. '/contactez-nous'
- Contact-us links now route properly in both languages
- Breadcrumbs are pulled from AEM for each page
- Looking For? Component pulled from AEM for Profile Page
- Default locale is set to 'und' undefined as no language data is provided, undefined defaults to /en/ when given a path

### CODE REVIEW

File: graphql/mappers/profile.js [hunk 1]
```diff
@@ -5,10 +5,37 @@ import { buildLink } from '../../lib/links'
...
+        }),
       pageName: response.data.schPagev1ByPath.item.scTitleEn,
       heading: response.data.schPagev1ByPath.item.scTitleEn,
+      lookingFor: {
+        title: enLookingForFragment.json[0].content[0].value,
+        subText: enLookingForFragment.json[1].content.map((element) => {
+          if (element.value) {
+            return element.value
+          }
+          return null
```
Comment (nibivi77) [inline graphql/mappers/profile.js:35]: This logic could be simplified to `return element.value || null` I think :thinking:

File: pages/contact-us.js [hunk 2]
```diff
@@ -8,9 +8,11 @@ import { getBetaBannerContent } from '../graphql/mappers/beta-banner-opt-out'
 import { getBetaPopupExitContent } from '../graphql/mappers/beta-popup-exit'
 import { getBetaPopupNotAvailableContent } from '../graphql/mappers/beta-popup-page-not-available'
 import logger from '../lib/logger'
+import { useRouter } from 'next/router'
 
 export default function ContactLanding(props) {
   const t = props.locale === 'en' ? en : fr
+  const router = useRouter()
```
Comment (nibivi77) [inline pages/contact-us.js:15]: Is this used?

File: package.json [hunk 3]
```diff
@@ -34,6 +34,7 @@
     "babel-plugin-macros": "^3.1.0",
     "markdown-to-jsx": "^7.1.8",
     "next": "^12.3.3",
+    "next-i18next": "^13.0.3",
```
Comment (nibivi77) [inline package.json:37]: Just wanted to check if this is used.

**Review by nibivi77 — COMMENTED** [review]
This is really good Curtis

File: package.json (hunk 3 above)
Comment (CurtisUnderwood) [reply package.json:37]: Yup, the i18n config in next.config.js allows us to set a default locale and define our used locales. Default locale is needed for the middleware.

**Review by nibivi77 — APPROVED** [review]
Awesome work Curtis!

**Pull request merged**